src/detect-bytetest.c

```diff
@@ -411,7 +411,7 @@ static DetectBytetestData *DetectBytetestParse(
             data->neg_op = true;
             op_ptr = &args[1][1];
             while (isspace((char)*op_ptr) || (*op_ptr == ',')) op_ptr++;
-            op_offset = op_ptr - &args[1][0];
+            op_offset = (uint32_t)(op_ptr - &args[1][0]);
```

Comment:
Should we have a debug statement before this `&args[1][0] < op_ptr`?

Reply:
`op_ptr = &args[1][1];` is the initialization of op_ptr (which is a not so readable way to say `&args[1][0] + 1`) , and then op_ptr only increases...
So, no need to have a debug statement, right ?

Reply:
Thanks for breaking it down. Indeed not needed. :)